.yamllint.yaml

```diff
@@ -8,3 +8,7 @@ rules:
   key-duplicates: enable
   truthy:
     allowed-values: ['true', 'false', 'yes', 'no', 'on']
+
+# Ignore template files from helm chart
+ignore:
+- Support/Multitenancy/Scalability-with-Fake-Nodes/ironic/templates/**/*.yaml
```

Comment:
Not so nice to just ignore them all. I know it can be hard to make templates pass as proper yaml though so I understand this.

Reply:
Thank you. I think yamllint isn't supposed to work with helm anyway. It will complain about go template syntax, as that's not proper yaml.